### ISSUE TITLE
[New Rule] Find consecutive slices

### ISSUE THREAD

**Issue by kasium** [MartinThoma/flake8-simplify#140]
## Explanation

Find consecutive slices which can be replaced by just one slice.

## Example

```python
foo = [1,2,3,4]

# Bad
foo[1:][:2]
foo[1:][2:]

# Good
foo[1:3]
foo[3:]
```
